README.md

```diff
@@ -1,5 +1,181 @@
+# Duke Robotics Club - RoboSub ROS
+
 ![](https://github.com/dukerobotics/robosub-ros/workflows/build/badge.svg)
 
-# Welcome to Duke Robotics Club
+We are a Duke University club competing in the RoboSub Competition. Check out [our website](https://duke-robotics.com).
+
+Our codebase is split into three repositories:
+
+1. This repo contains all of the required code for running and testing our robots.
+2. The [robosub-ros-docker](https://github.com/DukeRobotics/robosub-ros-docker) repo contains our Dockerfiles and instructions for how to build them.
+3. The [documentation](https://github.com/DukeRobotics/documentation) repo contains introductory projects, tutorials, and miscellaneous docs.
+
+Our code is Dockerized, making it straightforward to set up and run. See [Running Docker Images](#running-docker-images).
+
+Once the containers are up and running, go to [Running our Code](#running-our-code).
+
+
+## Software Stack
+
+Our stack is split into two containers: Onboard and Landside. Onboard holds all of the code that runs on the robot's internal computer. Landside holds all of the code that should be run on your local machine. They are connected via a local network and are configured to share ROS messages. For an in-depth explanation of the containers, see [robosub-ros-docker](https://github.com/DukeRobotics/robosub-ros-docker).
+
+Our code base is powered by the [Robot Operating System](https://www.ros.org) (ROS), and written primarily in Python.
```

Comment:
nit: codebase is one word I think

Reply:
Wikipedia says either is correct, but we tend to use codebase, so I'll change it.